libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryService.java

```diff
@@ -181,6 +179,16 @@ boolean isTelemetryReceiverRegistered() {
     return isTelemetryReceiverRegistered;
   }
 
+  boolean locationPermissionCheck() {
```

Comment:
since we've dependency on `supportlib-v4` is there a reason why we're not simply doing:
```suggestion
 private boolean locationPermissionCheck() {
    return ContextCompat.checkSelfPermission(getApplicationContext(), Manifest.permission.ACCESS_FINE_LOCATION)
      == PackageManager.PERMISSION_GRANTED;
  }
```